app/styles/_scroll-shadows-horizontal.less

```diff
@@ -0,0 +1,36 @@
+// Horizontal scroll drop shadows
+// Adds drop shadows to left and right edges that display when scrollable content is hidden
+// Test cases http://codepen.io/sg00dwin/pen/GWWRLR/
+@scroll-shadows-horizontal-bg:         rgba(255,255,255,1);
+@bg-color:                             @scroll-shadows-horizontal-bg;
+
+.scroll-shadows-horizontal(@bg-color: @scroll-shadows-horizontal-bg) {
```

Comment:
The application of .scroll-shadows-horizontal() is missing from .copy-to-clipboard.